README.md

```diff
@@ -117,9 +117,7 @@ If you clone this repository, use the checkout commands to access the sources fo
 | Name                                                                       | Keywords                  |
 | ---------------------------------------------------------------------------|---------------------------|
 | [Cockpit Sample Plugin](/cockpit/cockpit-sample-plugin)                    | Plugin                    |
-| [Failed Jobs Plugin](/cockpit/cockpit-failed-jobs-plugin)                  | Plugin                    |
-| [Javascript Only Plugin](/cockpit/js-only-plugin)                          | Plugin, Custom Script     |
-| [JQuery 3.4 Behavior Patch](/cockpit/jquery-34-behavior)                  | Plugin, Custom Script     |
```

Comment:
We link to the JQuery example from the update guide, we should update the link there: https://docs.camunda.org/manual/latest/update/minor/713-to-714/#update-to-jquery-3-5

Reply:
👍 I raised another PR to fix this: https://github.com/camunda/camunda-docs-manual/pull/653